internal/cli/server_upgrade.go

```diff
@@ -375,7 +381,7 @@ func (c *ServerUpgradeCommand) upgradeRunner(
 	}
 
 	s.Update("Runner found. Uninstalling previous runner...")
```

Comment:
hmmm yeah that makes sense. I was thinking we should be able to pull information necessary for upgrade from the context, but didn't think about the persistent storage (which I think we could also pull from the existing runner, but it would be a whole extra level of complexity.) Since we didn't handle the install for the runner, it's a good idea to let the user handle the update too.

Reply:
Yeah, I think this is definitely a case we should be able to handle. I think within the scope of runner installation/uninstallation right now, we should plan on getting to that in a future PR.